Zeeguu Reader/Pods/Zeeguu-API-iOS/ZeeguuAPI/ZeeguuAPI/ZeeguuAPI.swift

```diff
@@ -853,4 +874,55 @@ public class ZeeguuAPI {
 			completion(articles: nil)
 		}
 	}
+	
+	/// Retrieves a list of news feed that a user could start following.
+	///
+	/// - parameter language: The ID of the feed for which to retrieve a list of feed items.
+	/// - parameter completion: A block that will receive an array with the feed items.
+	public func getInterestingFeeds(language: String, completion: (feeds: [Feed]?) -> Void) {
+		let request = self.requestWithEndPoint(.GetInterestingFeeds, pathComponents: [language], method: .GET)
+		self.sendAsynchronousRequest(request) { (response, error) -> Void in
+			if let res = response, json = JSON.parse(res).array {
+				var feeds = [Feed]()
+				
+				for value in json {
+					if let title = value["title"].string, url = value["url"].string, id = value["id"].int, desc = value["description"].string, imURL = value["image_url"].string {
```

Comment:
By the way, wouldn't it make more sense to have these comments in the ZeeguuAPI iOS repository?